docs/actions.md

```diff
@@ -167,22 +167,27 @@ Rather than pass all the parameters to an action every time, you can bind certai
 
 ### Creating asynchronous actions
 
-JavaScript functions that continue execution in a callback function might need to return the activation result after the `main` function has returned. You can accomplish this using the `whisk.async()` and `whisk.done()` functions in your action.
+JavaScript functions that continue execution in a callback function might need to return the activation result after the `main` function has returned. You can accomplish this by returning a Promise in your action.
```

Comment:
@jasonpet When I merge this, I'm going to change this line to "JavaScript functions that run asynchronously may need to return the activation result after the `main` function has returned."  Let me know if you disagree.